recipe/build.sh

```diff
@@ -1,15 +1,20 @@
 #!/bin/bash
 set -ex
 
-if [[ ${gpu_variant:-} = "cuda-12" ]]; then
+if [[ ${gpu_variant:0:5} = "cuda-" ]]; then
     CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CUDA_ARCHITECTURES=all"
-    LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_CUBLAS=ON"
-    export LDFLAGS="$LDFLAGS -Wl,-rpath-link,/usr/lib64"
-elif [[ ${gpu_variant:-} = "cuda-11" ]]; then
-    CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_CUDA_ARCHITECTURES=all"
-    LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_CUBLAS=ON"
-    export CUDACXX=/usr/local/cuda/bin/nvcc
-    export CUDAHOSTCXX="${CXX}"
+    LLAMA_ARGS="${LLAMA_ARGS} -DLLAMA_CUDA=ON"
+    if [[ ${gpu_variant:-} = "cuda-11" ]]; then
+        export CUDACXX=/usr/local/cuda/bin/nvcc
+        export CUDAHOSTCXX="${CXX}"
+    else
+        LDFLAGS="$LDFLAGS -Wl,-rpath-link,${PREFIX}/lib/stubs/"
+        # TODO: This is a workaround. In the future, consider using cuda-compat instead of 
```

Comment:
What comment do you suggest? Is something like this ok?
```
# TODO: This is a workaround. In the future, consider using cuda-compat instead of 
# cuda-driver-dev. See: https://github.com/AnacondaRecipes/cuda-compat-feedstock
```

Reply:
Hmm yeah I think cuda-compat will be fine. It's just that in that case, llama.cpp will link against an older driver rather than stubs of a newer driver. see [description](https://github.com/conda-forge/cuda-compat-feedstock/blob/c47d7d9fc85d2f8b403f2a090a7a0dfe1fd957ae/recipe/meta.yaml#L40). But I think that will be fine